main.go

```diff
@@ -0,0 +1,55 @@
+package main
```

Comment:
We generally adopt the standard go style conventions for comments e.g. https://golang.org/doc/effective_go.html.

Packages should start with a comment of the form "Package xxx ....".
See for example https://golang.org/doc/effective_go.html#commentary.

There are some standard tools for checking this e.g. golint and go vet.
We typically enforce this via workflows on a repository
e.g. https://github.com/cloudspannerecosystem/harbourbridge/blob/master/.github/workflows/lint.yaml.

Reply:
@ankitmalikg2

Reply:
@rosspatil, @nevinheintze
 I have added package definition

Reply:
golint passes